e2e-tests/cypress/e2e/journeys/insurance/feedback/feedback-page-signed-in.spec.js

```diff
@@ -0,0 +1,37 @@
+import partials from '../../../partials';
+import { BUTTONS, PAGES } from '../../../../../content-strings';
+import { ROUTES } from '../../../../../constants';
+
+const CONTENT_STRINGS = PAGES.FEEDBACK_PAGE;
+
+const {
+  ALL_SECTIONS,
+  FEEDBACK,
+  ROOT,
+} = ROUTES.INSURANCE;
+
+context('Insurance - Feedback - Feedback page signed in', () => {
```

Comment:
The new tests have different copy for "signed in", can we align? Maybe "... X page - signed in"?